libs/game/info.ts

```diff
@@ -358,23 +358,49 @@ namespace info {
         player1.onScore(score, handler);
     }
 
+    /**
+     * Get the value of the current count down
+     */
+    //% block="countdown"
+    //% blockId=gamegetcountdown
+    //% weight=79
+    //% group="Countdown"
+    export function getCountdown() {
```

Comment:
Tiny nit i should have mentioned: for consistency with life / score / other getters should just be countdown(), not getCountdown()

Reply:
Thank you for the tips! They return 0, so I will do the same for countdown.

Reply:
Should I upload a new target to be played with?

Reply:
It seems pretty safe / I wouldn't bother personally. Changes lgtm, should be fine / can always fix if we notice an issue lster on while it's in beta